shared/src/common/enums/date/format-date.ts

```diff
@@ -0,0 +1,6 @@
+enum FormatDate {
+  DISTANCE = 'distance',
+  DASHES = 'dashes',
```

Comment:
We may have different formats for dash-separated date values, so let's name enum key as format to clarify the difference between templates
```suggestion
  YYYY_MM_DD = 'yyyy-MM-dd',
```
or just use union for `formatDate` type with used templates to get rid of enum keys naming challenge - eg
```
// in get-formatted-date
type FormatDate = 'distance' | 'yyyy-MM-dd' | ...etc;
```